packages/http-client-csharp/emitter/src/lib/client-model-builder.ts

```diff
@@ -34,7 +35,12 @@ export function createModel(sdkContext: SdkContext<NetEmitterOptions>): CodeMode
 
   navigateModels(sdkContext, modelMap, enumMap);
 
-  const rootApiVersions = getRootApiVersions(sdkPackage.clients);
+  const sdkApiVersionEnums = sdkPackage.enums.filter((e) => e.usage === UsageFlags.ApiVersionEnum);
+
+  const rootApiVersions =
+    sdkApiVersionEnums.length > 0
+      ? sdkApiVersionEnums[0].values.map((v) => v.value as string).flat()
+      : getRootApiVersions(sdkPackage.clients);
```

Comment:
If TCGC can guarantee that there will always be api version enums, then we can delete `getRootApiVersions` here.

Reply:
yes. I add this incase there is no such api version enum.